Makefile

```diff
@@ -6,7 +6,9 @@ ifeq ($(STATIC),1)
     STATIC_LIB=true
 endif
 
-CFLAGS=-O3 -DNDEBUG
+CC ?= cc
```

Comment:
GNU make defines CC, right?  Why was this added here but not in the Boron Makefile?

Reply:
Probably this redundant, just make sure that CC defined. Originally I made these changes for Gentoo Linux packaging faun and boron back in 2023 and don't quite remember my intentions :).